druid/src/app.rs

```diff
@@ -53,7 +53,7 @@ pub enum WindowSizePolicy {
 
 /// Window configuration that can be applied to a WindowBuilder, or to an existing WindowHandle.
 /// It does not include anything related to app data.
-#[derive(Debug, PartialEq)]
+#[derive(PartialEq)]
 pub struct WindowConfig {
```

Comment:
Ah, sorry, just noticed this: why no `Debug`?

Reply:
It was a `Debug` because `WindowLevel` was a `Debug`. And `WindowLevel` was a `Debug` just so we could warn "WindowBuilder::set_level({:?}) is not implemented". So basically once all the window levels are implemented we would not warn so we would not need `Debug`. 
Instead of saying ""WindowBuilder::set_level({:?}) is not implemented".. im saying "WindowLevel::DropDown and WindowLevel::Modal are not implemented in Windows".